src/pages/ProjectDetails/Rewards/List.tsx

```diff
@@ -47,7 +47,10 @@ const RewardList: React.FC = () => {
   });
 
   const rewards = data?.pages.flatMap(page => page.rewards) || [];
-  const isRewardDisabled = projectBudget?.remainingDollarsEquivalent === 0 || rewards.length === 0;
+  const isRewardDisabled =
+    !projectBudget?.remainingDollarsEquivalent ||
+    projectBudget?.remainingDollarsEquivalent === 0 ||
+    rewards.length === 0;
```

Comment:
This is the same isn't it ?
```suggestion
  const isRewardDisabled = !projectBudget?.remainingDollarsEquivalent || rewards.length === 0;
```

Reply:
yes we can do that my bad

Reply:
fixed !